Source/WebKit/UIProcess/Cocoa/AuxiliaryProcessProxyCocoa.mm

```diff
@@ -155,4 +155,11 @@
 }
 #endif
 
+void AuxiliaryProcessProxy::requestRemoteProcessTermination()
+{
+    callOnMainRunLoop([protectedThis = Ref { *this }]() {
```

Comment:
This needs to be dealt with at the call site, not here. AuxiliaryProcessProxy is not a thread-safe class, we shouldn't be calling its member functions on a non-main thread.

Reply:
Ah, good catch, fixed!